templates/docs/examples/patterns/side-navigation/toggle.html

```diff
@@ -0,0 +1,67 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Side navigation / Mobile toggle{% endblock %}
+
+{% block standalone_css %}patterns_side-navigation{% endblock %}
+
+{% block content %}
+<div class="p-notification--caution u-hide--small">
```

Comment:
@bartaz why the notification in the example btw? why not have the copy in the md file

Reply:
I thought it could be useful to have it in the example to make sure the message is shown not only in documentation text, but also if example is opened directly from `Examples` page.

I guess before we usually add notes in docs instead, so I can move it out if we feel it obstructs the example itself.